.github/workflows/e2e.yml

```diff
@@ -40,8 +40,15 @@ jobs:
       -
         name: Update daemon.json
         run: |
-          sudo jq '.experimental = true' < /etc/docker/daemon.json > /tmp/docker.json
-          sudo mv /tmp/docker.json /etc/docker/daemon.json
+          if [ ! -f /tmp/foo.txt ]; then
```

Comment:
?

Reply:
LOOOOOL 🙈 🙈🙈 copy/pasta because I always forget the correct one, so I tried first

Reply:
Fixed now; only now wondering; would it need `sudo` to check for the file? 🤔 I guess the path would be _readable_ at least for non-root, so probably it's fine as-is.